app/controllers/languages_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# LanguagesController: for language functionality
+class LanguagesController < ApplicationController
+  def index; end
```

Comment:
Do you actually need this?

Also, I'd prefer people have to authenticate to this, make sure to add `before_action :authenticate_user!` to the top of the controller.

Reply:
:+1: authentication

Reply:
Nope! It was auto-generated by rails, and i wasnt sure if it was needed. Removing, and adding authentication.